plugins/psx/src/psx-plugin.vala

```diff
@@ -0,0 +1,32 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private class Games.Psx : Object, Plugin {
+	private const string FINGERPRINT_PREFIX = "psx";
```

Comment:
Maybe we should name it 'playstation' rather than 'psx': it's longer but ore explicit. The same goes for any place where 'psx' is used, even the directory.